index.html

```diff
@@ -925,99 +925,98 @@ <h2>The <code>touch-action</code> CSS property</h2>
                 <tr><th>Computed value:</th><td>Same as specified value.</td></tr>
             </table>
 
-            <p>The <code>touch-action</code> CSS property determines whether touch input MAY trigger the user agent's default behavior. This includes, but is not limited to, behaviors such as panning or zooming. See the section on <a href="#details-of-touch-action-values"><code>touch-action</code> values</a>.</p>
+            <p>The <code>touch-action</code> CSS property determines whether <a>direct manipulation</a> interactions (which are not limited to touch pointers, despite the property's name) MAY trigger the user agent's scrolling and zooming behavior. See the section on <a href="#details-of-touch-action-values"><code>touch-action</code> values</a>.</p>
```

Comment:
touch pointer? That sounds a bit odd and not used elsewhere, I think

Reply:
yup, good point. in my head, it was along the lines of "`touch` pointers" (as in the `pointerType`), but it reads awkward, you're right

Reply:
changed to just "touch"